functionary/ui/templates/account/login.html

```diff
@@ -37,105 +33,86 @@
-                                        <div class="columns is-multiline is-mobile">
+                            <div class="card-footer text-center bg-body">
+                                <button class="btn btn-success text-light" form="loginForm" type="submit">Sign In</button>
+                            </div>
+                        </div>
+                    {% endblock content %}
+                    {% block extracontent %}
+                        {% configured_providers as socialaccount_providers %}
+                        {% if socialaccount_providers %}
+                            <div class="card shadow text-primary bg-body">
+                                <div class="card-body">
+                                    <h5 class="card-title text-center">Or login with:</h5>
+                                    <div class="container text-center">
+                                        <div class="row">
                                             {% for provider_pair in socialaccount_providers %}
                                                 {% if provider_pair.provider.id == "openid" %}
                                                     {% for brand in provider_pair.provider.get_brands %}
-                                                        <div class="column">
+                                                        <div class="col">
+                                                            {% csrf_token %}
                                                             {% comment %}The double class on here will default to openid's icon if the brand doesn't have one{% endcomment %}
                                                             <button title="{{ brand.name }}"
-                                                                    class="button icon-{{ provider_pair.provider.id }} icon-{{ brand.id }}"
-                                                                    type="submit"
+                                                                    class="btn icon-{{ provider_pair.provider.id }} icon-{{ brand.id }}"
+                                                                    type="button"
                                                                     hx-get="{% provider_login_url provider_pair.provider.id openid=brand.openid_url process=process %}">
-                                                                &nbsp;{{ brand.name }}
+                                                                {{ brand.name }}
                                                             </button>
                                                         </div>
                                                     {% endfor %}
                                                 {% else %}
-                                                    <div class="column">
+                                                    <div class="col">
+                                                        {% csrf_token %}
```

Comment:
Just curious, is this actually sending the CSRF token with the login with <account> request?